burger-app/src/components/Main/Burger/index.js

```diff
@@ -1,13 +1,37 @@
 import styled from "styled-components";
 import Image from "../image";
 
-const Burger = ({ orderPrice }) => {
+const Burger = ({ orderPrice, orderIngredients, ingredientsInOrder, checkout }) => {
   return (
     <BurgerWrapperStyled>
       <TotalPriceStyled>Total: {orderPrice}₴</TotalPriceStyled>
       <Image ingredient="top_bun" alt="Top Bun" />
-      <SpanStyled>Please, start by adding products...</SpanStyled>
+      {orderIngredients ? (
+        <SpanStyled />
+      ) : (
+        <SpanStyled>Please, start by adding products...</SpanStyled>
+      )}
+      <IngredientWrapperStyled>
+      {orderIngredients ? (
+        ingredientsInOrder.map((element, index) => {
+          return (
+            <IngredientsStyled
+              key={element + index}
+              src={require(`../../../../assets/images/updateBurgerImg/${element}.png`)}
+              alt={element}
+              style={{
+                bottom: 300 + index * 10,
+                zIndex: index + 1,
+              }}
+            />
+          );
+        })
+      ) : (
+        <SpanStyled />
+      )}
+      </IngredientWrapperStyled>
       <Image ingredient="bottom_bun" alt="Bottom Bun" />
```

Comment:
It's jumping when a user starts adding ingredients


https://user-images.githubusercontent.com/40695369/212707137-5767fc52-f3fc-4f8a-a1db-7f525efd782c.mov